docs/UserGuide.md

```diff
@@ -82,19 +82,18 @@ TaskTracker makes time-management smart and simple for you!
 6. You're good to go! Try adding your first Task. Press Enter to enter a
    command. For adding a floating task, try:
 
-	* `add "Learn Task Tracker"`.
+	* `add "Learn Task Tracker" p-5`.
 
     ![alt text](./images/userguide/1.png "command bar")
 
     Let's do another one:
 
-	* `add "bake potatoes" p-2`
+	* `add "bake potatoes"`
 
     ![alt text](./images/userguide/2.png "")
 
     Notice how `bake potatoes` is below `Learn Task Tracker`. That's because
-    `bake potatoes` is of number 2 priority, while `Learn Task Tracker` with an
-    undefined priority takes the default highest priority number of 0.
+    `bake potatoes` is of default priority 0, lower than `Learn Task Tracker` priority of 5.
```

Comment:
For this part, I think the picture of the mock ui should be changed as soon as possible, or we will just forget it very soon.
Just as a suggestion, why don't you just revise this documentation according to all the pictures? So that the pictures  are still usable at this time.